### ISSUE TITLE
FYI: https://github.com/JOJ0/synadm

### ISSUE THREAD

**Issue by eMPee584** [KnugiHK/synapse-admin-api-python#2]
.. just had a look at current status of synapse admin tools (~medium impressive xD) and found a few, amongst which I like yours and https://github.com/JOJ0/synadm which already seems to have quite the endpoint coverage (haven't tried either yet). So this issue is just to let you know synadm is out there, maybe there's some synergy potential :cowboy_hat_face:

**Comment by KnugiHK** [comment]
Yeah, I also found JOJ0/synadm earlier too.

When it comes to the coverage of endpoints, as far as I observed, my repo implemented more endpoints than synadm. However, the aims and objectives of two repos are not the same (mine is a wrapper, synadm is a cli tool). I will look for some potential collaborations between two repos. If needed, I will contact JOJ0.

Glad that you found interested in this repo and thanks for your information!

**Comment by eMPee584** [comment]
Ah cool, that's good to hear that you're already aware of the other project! Often times, people with similar needs cross the pain-counter-action threshold at approximately the same time and indulge in duplicate effort.. Even due diligence (to do some prior implementation research) doesn't necessarily prevent that.
My estimation of API coverage was purely from the impressive list on the other project's readme, so that obviously is not a reliable metric 😅
so thanks for the constructive response and best regards from 🇪🇺 to 🇭🇰

**Comment by KnugiHK** [comment]
Haha, true. I started the idea of making a wrapper on 13 Oct 2020. At that moment, no other tools are available that suit my needs, therefore, I decided to do it myself. However, my first commit on actual code is on 8 Jan 2021. And synadm is out on 22 Oct 2020, so, that's all about timing I guess.
Btw, I am developing a Telegram bot based on this wrapper, if you are interested also, check my GitHub profile later! Thanks!
